starter-code/pizza.js

```diff
@@ -1 +1,92 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+  $("li:contains(white sauce)").toggle();
+  $(".btn-sauce").toggleClass("active");
+  $(".btn-crust").toggleClass("active");
+  $("li:contains(gluten-free crust)").toggle();
+
+    var price = $('strong:contains($21)');
+    price.html("$13");
+
+
+
+
+
+    $(".btn-pepperonni").click(function(event) {
+      if ($(".btn-pepperonni").prop('class') === "btn btn-pepperonni active") {
```

Comment:
Resourceful. I would just use `$(".btn-pepperonni").hasClass("active") ` in my if statement, which is either going to return true or false, making it cleaner. Finding and learning the right methods for the right situation comes with more experience (and googling).